protractor/e2e/integration/second-page.spec.ts

```diff
@@ -61,13 +61,28 @@ describe('tests for second page', async () => {
             expect(await secondPage.infoSubmitButton.isDisplayed()).toBeTruthy();
             await secondPage.infoSubmitButton.click();
 
+    });
 
 
+    fit('check the blog (homework)', async () => {
 
+        //await browser.get('https://ng-book.com/2/');
 
-    });
-
-
+        await secondPage.menuItemBlog.click();
+        expect(await browser.getCurrentUrl()).toContain('https://blog.ng-book.com/');
+        expect(await secondPage.header.count()).toEqual(20);
+        expect(await secondPage.description.count()).toEqual(20);
+        expect(await secondPage.link.count()).toEqual(20);
+        expect(await secondPage.linkFirstPage.isDisplayed()).toBeTruthy();
+        expect(await secondPage.linkOne.isDisplayed()).toBeTruthy();
+        expect(await secondPage.linkTwo.isDisplayed()).toBeTruthy();
+        expect(await secondPage.linkNext.isDisplayed()).toBeTruthy();
+        expect(await secondPage.linkLastPage.isDisplayed()).toBeTruthy();
+        await secondPage.linkTwo.click();
+        expect(await secondPage.getCurrentUrl()).toContain('/page/2/');
+        expect(await secondPage.blockPost.isDisplayed()).toBeTruthy();
+        expect(await secondPage.blockPost.get(0).getText());
 
+    });
```

Comment:
It's advisable to divide all this logic into 2 it blocks:
The first one for checking "Navigate to blog and check articles, pages navigation"
And the second one: "Navigate to blog and check articles, pages navigation"